### PR TITLE
Fix: Rename util.xqm to eutil.xqm

### PR DESCRIPTION
**Pull request by riedde**
After we decided multiple times to fix the name of the module for consistency reasons, I fixed it.

### CODE REVIEW

**Comment by bwbohl** [comment]
I generally approve but think this shoud wait until other PRs are resolved

**Comment by peterstadler** [comment]
I did not take part in the discussion but is it right that the namespace stays "http://www.edirom.de/xquery/util" rather than "http://www.edirom.de/xquery/eutil"?

**Comment by riedde** [comment]
I would prefer to also update the namespace. Does this has further consequences? I think this module is only used inside edirom, so the change would be as breaking as renaming the module, but not more.

**Comment by riedde** [comment]
@daniel-jettka Done, Thanks for your comment!

File: add/controller.xql [hunk 1]
```diff
@@ -3,7 +3,7 @@ xquery version "3.1";
 (: IMPORTS ================================================================= :)
 
 import module namespace edition="http://www.edirom.de/xquery/edition" at "data/xqm/edition.xqm";
-import module namespace eutil = "http://www.edirom.de/xquery/util" at "data/xqm/util.xqm";
+import module namespace eutil = "http://www.edirom.de/xquery/eutil" at "data/xqm/util.xqm";
```
Comment (daniel-jettka) [inline add/controller.xql:6]: should this also be data/xqm/eutil.xqm ?

File: testing/XQSuite/eutil-tests.xqm [hunk 2]
```diff
@@ -2,7 +2,7 @@ xquery version "3.1";
 
 module namespace eut = "http://www.edirom.de/xquery/xqsuite/eutil-tests";
 
-import module namespace eutil = "http://www.edirom.de/xquery/util" at "xmldb:exist:///db/apps/Edirom-Online/data/xqm/util.xqm";
+import module namespace eutil = "http://www.edirom.de/xquery/eutil" at "xmldb:exist:///db/apps/Edirom-Online/data/xqm/util.xqm";
```
Comment (daniel-jettka) [inline testing/XQSuite/eutil-tests.xqm:5]: should this also be .../data/xqm/eutil.xqm ?

**Review by daniel-jettka — CHANGES_REQUESTED** [review]
see inline comments

File: testing/XQSuite/eutil-tests.xqm (hunk 2 above)
Comment (riedde) [reply testing/XQSuite/eutil-tests.xqm:5]: I have never been to this place before, but yes, It won't work this way
```suggestion
import module namespace eutil = "http://www.edirom.de/xquery/eutil" at "xmldb:exist:///db/apps/Edirom-Online/data/xqm/eutil.xqm";
```

File: add/controller.xql (hunk 1 above)
Comment (riedde) [inline add/controller.xql:6]:
```suggestion
import module namespace eutil = "http://www.edirom.de/xquery/eutil" at "data/xqm/eutil.xqm";
```

**Review by daniel-jettka — APPROVED** [review]

**Review by daniel-jettka — APPROVED** [review]

**Review by hizclick — APPROVED** [review]
Tested on ExampleEdition

**Pull request merged**